### PR TITLE
Add documentation for diagrams' tools

### PR DESCRIPTION
**Pull request by pblottiere**
Since recently, tools for labels are fully usable with diagrams too : highlight, move, hide/show and pin/unpin.

Thus, this PR add documentation for these features.

Moreover, some whitespaces has been removed by my editor.

Note that this feature is for 2.16.

### CODE REVIEW

**Comment by yjacolin** [comment]
@pblottiere could you add the ticket number of the QGIS doc repository? Thanks.

**Comment by pblottiere** [comment]
@yjacolin I'm sorry i don't see any ticket related to these features. I missed it?

**Comment by yjacolin** [comment]
did you add the FEATURE label in your commit to the QGIS repository? if so it should have a ticket, if not could you add it in the commit message  for your next commit ;) Here is an example: https://github.com/qgis/QGIS/commit/99210ecfd713f4380125c4ccfa0908142c8ee3a9

**Comment by pblottiere** [comment]
OK, thanks for the tip! I'll do that the next time :)

**Comment by yjacolin** [comment]
This PR need to be rebased.

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -1842,6 +1867,47 @@ The size and the attributes can also be an expression.
...
+
+.. _figure_diagrams_6:
+
+.. only:: html
+
+   **Figure Diagrams 6:**
+
+.. figure:: /static/user_manual/working_with_vector/diagram_toolbar.png
+   :align: center
+   :width: 15em
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: disclaimer: didn't build the html to check how it renders.
Wonder why width variable as needed for this figure

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -1842,6 +1867,47 @@ The size and the attributes can also be an expression.
...
+
+   Diagram/Label toolbar
+
+These tools allow to specifically work on each diagram to:
+
+* |mActionPinLabels| :sup:`Pin/Unpin Labels And Diagrams` that has data-defined
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: We get rid of mAction... in substitutions so rather use |pinLabels|

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 3]
```diff
@@ -1842,6 +1867,47 @@ The size and the attributes can also be an expression.
...
+  position. By clicking or draging an area, you pin diagram(s). If you click or
+  drag an area holding ``Shift``, diagram(s) are unpinned. Finally, you can
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: I think it's better to use :kbd:\`Shift\` according to [guidelines](http://docs.qgis.org/2.14/en/docs/documentation_guidelines/writing.html)

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 4]
```diff
@@ -1842,6 +1867,47 @@ The size and the attributes can also be an expression.
...
+  also click or drag an area holding ``Ctl`` to toggle the pin status of
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Ctl or Ctrl?
with :kbd: as above

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 5]
```diff
@@ -1842,6 +1867,47 @@ The size and the attributes can also be an expression.
...
+  diagram(s).
+* |mActionShowPinnedLabels| :sup:`Highlight Pinned Labels and DIagrams`. If the
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: **A**nd D**i**agrams

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 6]
```diff
@@ -1842,6 +1867,47 @@ The size and the attributes can also be an expression.
...
+  vector layer of the diagram is editable, then the highlighting is green,
+  otherwise it's blue.
+* |mActionMoveLabel| :sup:`Move Label And Diagram` that has data-defined
+  position. You just have to drag the diagram to the desired place.
+* |mActionShowHideLabels| :sup:`Show/Hide Labels and Diagrams` that has
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: **A**nd

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 7]
```diff
@@ -1842,6 +1867,47 @@ The size and the attributes can also be an expression.
...
+  data-defined visbility. If you click or drag an area holding ``Shift``,
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: visibility
:kbd:\`Shift\`

**Comment by pblottiere** [comment]
I rebased and fixed the documentation according to your comments.

**Comment by yjacolin** [comment]
thanks :)

**Pull request merged**